src/main/java/org/jabref/gui/sharelatex/ShareLatexLoginDialogView.java

```diff
@@ -27,6 +28,10 @@
     private SharelatexConnectionProperties props;
     private ShareLatexLoginDialogViewModel viewModel;
 
```

Comment:
You need to put this in the ctor:

   ViewLoader.view(this)
                  .load()
                  .setAsDialogPane(this);